src/test/kotlin/GenMutatorTest.kt

```diff
@@ -1,43 +1,24 @@
-
 import backend.GenMutator
 import backend.config.Config
-import backend.model.Gen
-import backend.model.Genome
+import backend.model.Gen.*
 import io.kotest.core.spec.style.FunSpec
-import io.kotest.matchers.collections.shouldContain
 import io.kotest.matchers.shouldBe
 
 class GenMutatorTest : FunSpec({
-
   test("combine without switching") {
-    val config = Config.test.copy(minMutations = 2, maxMutations = 3)
-    val mutator = GenMutator(config)
-    val genome1 = Genome.random(config.genomeLength)
-    val genome2 = Genome.random(config.genomeLength)
-
-    mutator.combine(genome1, genome2, 0.5).genes.size shouldBe config.genomeLength
-
-    val combinedGenom = mutator.combine(genome1, genome2, 1.0)
-    combinedGenom.genes.size shouldBe config.genomeLength
-    var differences = 0
-    (0..<config.genomeLength).forEach {
-      if (genome1.genes[it] != combinedGenom.genes[it]) differences++
-    }
-    (config.minMutations..config.maxMutations) shouldContain differences
+    val mutator = GenMutator(Config.test.copy(maxMutations = 2))
+    mutator.combine(mutator.random(), mutator.random(), 0.5).genes shouldBe
```

Comment:
why? using `random` isn't descriptive :/

Reply:
do not rely on the seed in tests. It was introduced only to make testing possible, not seed-driven.
According to the `kotlin.random.Random` docs
`Note: Future versions of Kotlin may change the algorithm of this seeded number generator so that it will return a sequence of values different from the current one for a given seed.`

Reply:
Yes, I am looking for a better way to test this but every one I find has drawbacks